src/Export/PluginSettings.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Towa\GdprPlugin\Export;
+
+use Towa\GdprPlugin\Acf\AcfCookies;
+use Towa\GdprPlugin\Acf\AcfSettings;
+use Towa\GdprPlugin\Helper\PluginHelper;
+use Towa\GdprPlugin\Plugin;
+use Towa\GdprPlugin\SettingsTableAdapter;
+
+// phpcs:disable PSR1.Files.SideEffects
+if (!defined('ABSPATH')) {
+    exit; // Exit if accessed directly.
+}
+// phpcs:enable
+
+/**
+ * Gets all plugin settings as well as Acf settings in all Languages
+ */
+class PluginSettings
+{
+    /**
+     * All Acf Settings in all Languages.
+     *
+     * @var array
+     */
+    public $acfSettings;
+
+    /**
+     * All Settings saved in the plugin settings table.
+     *
+     * @var array
+     */
+    public $pluginSettings;
+
+    public function __construct()
+    {
+        $this->acfSettings = $this->getAcfSettings();
+        $this->pluginSettings = $this->getPluginSettings();
+    }
+
+    /**
+     * Get all Acf settings, in all languages.
+     */
+    private function getAcfSettings(): array
```

Comment:
i am not happy about a getter which is called like the property but is returning something different, maybe you should remove the constructor  implement [JsonSerializable](https://www.php.net/manual/en/jsonserializable.jsonserialize.php)

Reply:
i refactored the class a lot. I think its cleaner now. Also more reuseable for the import.